assets/helpers/internationalisation/country.js

```diff
@@ -134,8 +134,7 @@ function fromGeolocation(): ?IsoCountry {
 function detect(): IsoCountry {
   const country = fromPath() || fromQueryParameter() || fromCookie() || fromGeolocation() || 'GB';
   // cookie.set('GU_country', country, 7);
```

Comment:
This line should be uncommented too, so that we remember the country selection across pages

Reply:
@JustinPinner and I discussed this. For the moment we're passing this between pages using the query param. We thought this was safer than assigning a cookie, because at the moment there's no way for the user to change their country once they get assigned.